d2l-scroll-wrapper.html

```diff
@@ -125,6 +125,7 @@
 
 			attached: function() {
 				this.async(this.checkScrollbar);
+				this.async(this.checkScrollbar);
```

Comment:
In terms of perceived performance, I don't see an impact in any of the other standard browsers (Edge, Chrome, Safari, Firefox). Also, the issue seems to be resolved on IE11. However, there is a delay in rendering the page on IE11. This fix is better than before, therefore LGTM!